srcpkgs/sof-bin/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'sof-bin'
+pkgname=sof-bin
+version=1.5.1
+revision=1
+archs="i686 x86_64"
+wrksrc=${pkgname}-stable-v${version}
+make_dirs="/usr/lib/firmware/intel/sof/intel-signed 0755 root root
+ /usr/lib/firmware/intel/sof/public-signed 0755 root root
+ /usr/lib/firmware/intel/sof-tplg 0755 root root"
```

Comment:
These directories already exist in the tarball. `make_dirs` is for empty directories that are cleaned when generating the tarball, and are instead created at package installation time.

Reply:
Ah, I misunderstood what the `make_dirs` variable was for. It's been removed.